pkg/vere/main.c

```diff
@@ -2315,6 +2321,45 @@ _cw_play_impl(c3_d eve_d, c3_d sap_d, c3_o mel_o, c3_o sof_o, c3_o ful_o)
   return pay_d;
 }
 
+/* _cw_play_fork(): spawn a subprocess for event replay.
+*/
+static c3_i
+_cw_play_fork(c3_d eve_d, c3_d sap_d, c3_o mel_o, c3_o sof_o, c3_o ful_o)
+{
+  pid_t pid;
+  c3_i sat_i;
+  c3_c eve_c[21], sap_c[21] = { 0 };
+  sprintf(eve_c, "%" PRIu64, eve_d);
+  sprintf(sap_c, "%" PRIu64, sap_d);
+  // if ( 0 < sprintf(eve_c, "%" PRIu64, eve_d) ||
+  //      0 < sprintf(sap_c, "%" PRIu64, sap_d) )
+  // {
+  //   fprintf(stderr, "play: error parsing args\r\n");
+  //   return 1;
+  // }
+  c3_c *argv[] = { 
+    u3_Host.wrk_c, "play", u3_Host.dir_c, 
+    "--replay-to", eve_c,
+    "--snap-at", sap_c, 
+    _(mel_o) ? "--auto-meld" : 0,  //
+    _(sof_o) ? "--soft-mugs" : 0,  //
+    _(ful_o) ? "--full" : 0,       //  XX
```

Comment:
@joemfb is there a better way to add binary flags to `argv` so that we don't ever insert extra nulls?